### ISSUE TITLE
[VERY IMPORTANT] Holoparasites can lose their master without dying, making them immortal

### ISSUE THREAD

**Issue by fludd12** [ParadiseSS13/Paradise#3091]
Welcome to hell!
Welcome to hell!
Welcome to hell!

**Comment by Fox-McCloud** [comment]
Steps on how to reproduce this would be nice.

**Comment by scrubmcnoob** [comment]
I am the holoparasite he is probably talking about.
My master and I were fighting in the med bay. His death was not one of being robusted to death but rather he was blown up while I was outside him. From there I realized he was just giblets and I was still alive. I then stopped fighting thinking it was lag to why I was still alive but then realized I was just alive. So went afk for dinner and came back to realize I am invincible. So I just started inhabiting a box and being docile.

So in short, master was gibbed while I was outside him.

**Comment by Regen1** [comment]
No idea how this happened
http://puu.sh/mhs3r/56661ca2de.png

**Comment by scrubmcnoob** [comment]
Ye after my master went boom, I became independent. I then decided a steady life in a box was the way to live.

**Comment by fludd12** [comment]
It's happened again recently, with completely immortal ranged attacker going on rampage forcing a shuttle call.

I still have no clue how this stuff happens.

**Comment by Mijhead** [comment]
http://jojo.wikia.com/wiki/Notorious_B.I.G.
